influx-writer/influx-writer.py

```diff
@@ -36,6 +36,12 @@ def __init__(self, ls, influx, redis):
         self.inserts = 0
         self.time = 0
 
+    @staticmethod
```

Comment:
We already have this somewhere -- lets not duplicate this and put this into a common module please.